### PR TITLE
fix!: updated postgresql and removed manual secret copying

### LINKED ISSUE #28: Pull out dev namespace and manual secret copying
Because we're using the most up-to-date postgresql UDS package, we don't need to do the manual secret copy step anymore.

### PR DESCRIPTION
**Pull request by JoeHCQ1**
Closes #28

### CODE REVIEW

**Comment by JoeHCQ1** [comment]
Reviewers, the upgrade step fails on bafd88c4e1f6baf6931d8c9ceb46d42a7f1fbe1d. I do not believe this should prevent merging because:

1. We have no (known) users of this package.
2. The error would be fairly easy to solve if necessary, the old version of this package put the secret in one-place, and when upgrading, it'd appear the operator does not generate a new secret, therefore, confluence never comes up and is stuck with a config error.

LTMK if this needs to be fixed. I think the simplest approach would be to leave the manual secret copy in, only modify it to copy from the old secret if needed, and then back that out in a subsequent PR.

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -58,6 +58,13 @@ jobs:
           flavor: ${{ matrix.flavor }}
           type: ${{ matrix.type }}
 
+      - name: Debug Cluster
```
Comment (corang) [inline .github/workflows/test.yaml:61]: should be removed before merge to main

File: .gitignore [hunk 2]
```diff
@@ -26,3 +26,5 @@ test/tf/public-ec2-instance/.terraform
 terraform.tfstate
 terraform.tfstate.backup
 .terraform.lock.hcl
+
+confluence_license.txt
```
Comment (corang) [inline .gitignore:30]: What's this for?

File: bundle/uds-bundle.yaml [hunk 3]
```diff
@@ -7,13 +7,14 @@ metadata:
...
 
 packages:
+  # Creates the confluence namespace so the postgres-operator can create the secret there.
   - name: dev-namespace
     path: ../
     ref: 0.1.0
 
   - name: postgres-operator
     repository: ghcr.io/defenseunicorns/packages/uds/postgres-operator
-    ref: 1.10.1-uds.4-upstream
```
Comment (corang) [inline bundle/uds-bundle.yaml:16]: We usually let renovate do this, if it's not picking it up we need to check our renovate config

File: common/zarf.yaml [hunk 4]
```diff
@@ -7,6 +7,15 @@ metadata:
 components:
   - name: confluence
     required: true
+    actions:
```
Comment (corang) [inline common/zarf.yaml:10]: This only works when we're using the postgres operator package from uds, there are a bunch of other possible deployments of this, e.g. using RDS

**Review by corang — CHANGES_REQUESTED** [review]
I'm pretty sure all of this will be superseded by https://github.com/defenseunicorns/uds-package-confluence/pull/34 since it's such a large change to move to the upstream chart. Unless I missed something this can most likely be closed

**Comment by JoeHCQ1** [comment]
OBE

**Pull request closed without merging**